FinniversKit/Sources/Components/SelectionBoxes/Selectionbox.swift

```diff
@@ -149,18 +155,24 @@ extension Selectionbox {
     // MARK: Private methods
 
     private func setupSubviews() {
-        addSubview(titleLabel)
-        addSubview(stack)
+        
+        addSubview(containerStack)
+        
+        containerStack.addArrangedSubviews([
+            titleLabel,
+            stack
+        ])
+        
+        if titleLabel.text == nil {
```

Comment:
If you set the label to hidden the stack view will hide it and adjust spacing automatically.